config/config.go

```diff
@@ -163,31 +169,12 @@ func (c *Config) Save() error {
 
 // Set preserve sd-local config with new value.
 func (e *Entry) Set(key, value string) error {
-	switch key {
-	case "api-url":
-		e.APIURL = value
-	case "store-url":
-		e.StoreURL = value
-	case "token":
-		e.Token = value
-	case "launcher-version":
-		if value == "" {
-			value = "stable"
-		}
-		e.Launcher.Version = value
-	case "launcher-image":
-		if value == "" {
-			value = "screwdrivercd/launcher"
-		}
-		e.Launcher.Image = value
-	case "uuid":
-		if value == "" {
-			value = "-"
-		}
-		e.UUID = value
-	default:
+	var m map[string]interface{}
+	mapstructure.Decode(e, &m)
```

Comment:
Do you need to check error here?

Reply:
OK.
I will add error handling.

However, I don't know in what case the error occurs.